### PR TITLE
Update wording around output_file()

### PR DESCRIPTION
**Pull request by tcmetzger**
This PR updates some wording regarding `output_file()` in various places. It also adds `output_file()` and `output_notebook()` to rst_epilog.txt. This work is in preparation of making documentation example names more consistent (#10377).

### CODE REVIEW

File: bokeh/io/output.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def output_file(filename: PathLike, title: str = "Bokeh Plot",
 def output_notebook(resources: Resources | None = None, verbose: bool = False,
         hide_banner: bool = False, load_timeout: int = 5000, notebook_type: NotebookType = "jupyter") -> None:
     ''' Configure the default output state to generate output in notebook cells
-    when :func:`show` is called. Note that, :func:`show` may be called multiple
+    when :func:`show` is called. Note that :func:`show` may be called multiple
```
Comment (bryevdv) [inline bokeh/io/output.py:94]: could these be `|show|` and `|save|` now?

File: sphinx/source/docs/first_steps/first_steps_7.rst [hunk 2]
```diff
@@ -14,22 +14,28 @@ visualizations.
...
 Bokeh creates the HTML file when you call the |show| function. This function
 also automatically opens a web browser to display the HTML file.
 
 If you want Bokeh to only generate the file but not open it in a web browser,
-use the |save| function instead. You need to import the ``save()`` function
-before using it, just like you did for ``show()``.
+use the |save| function instead.
+
+You need to import the ``save()`` and  ``output_file()`` functions before using,
+just like you did for ``show()``.
```
Comment (bryevdv) [inline sphinx/source/docs/first_steps/first_steps_7.rst:38]: same as above, `|show|` etc?

File: bokeh/io/output.py (hunk 1 above)
Comment (tcmetzger) [reply bokeh/io/output.py:94]: totally!

File: sphinx/source/docs/first_steps/first_steps_7.rst (hunk 2 above)
Comment (tcmetzger) [reply sphinx/source/docs/first_steps/first_steps_7.rst:38]: Sure!

**Review by bryevdv — APPROVED** [review]

**Pull request merged**